iBridges.py

```diff
@@ -186,9 +186,9 @@ def login_function(self):
             self.setCursor(PyQt6.QtGui.QCursor(PyQt6.QtCore.Qt.CursorShape.ArrowCursor))
             return
         except Exception as unknown:
```

Comment:
For concistency's sake, `e` should be `error`, I think.

Reply:
Consistency is nice.